Tasks/AzureFunctionAppV2/operations/AzureAppServiceUtility.ts

```diff
@@ -122,9 +122,18 @@ export class AzureAppServiceUtility {
 
     public async getKuduService(): Promise<Kudu> {
         var publishingCredentials = await this._appService.getPublishingCredentials();
+        var scmPolicyCheck = await this.isSitePublishingCredentialsEnabled();            
+        
+
         if(publishingCredentials.properties["scmUri"]) {
-            tl.setVariable(`AZURE_APP_SERVICE_KUDU_${this._appService.getSlot()}_PASSWORD`, publishingCredentials.properties["publishingPassword"], true);
-            return new Kudu(publishingCredentials.properties["scmUri"], publishingCredentials.properties["publishingUserName"], publishingCredentials.properties["publishingPassword"]);
+                    if(!scmPolicyCheck) {
+                        var accessToken = await this._appService._client.getCredentials().getToken();
+                    }
+                    else{
```

Comment:
should we instrument this call to find the # using basic or bearer auth from ADO telemetry or is this something can be obtained from Kudu end ?

Reply:
Hello, we can add a log here yes.